antenna/app.py

```diff
@@ -144,107 +50,114 @@ def build_config_manager():
     return config
 
 
-class AppConfig(RequiredConfigMixin):
```

Comment:
I merged the configuration into the app class so we don't have a separate thing that added a level of configuration indirection and was confusing.